tests/test_ell1h.py

```diff
@@ -143,6 +144,17 @@ def test_J0613_STIG(toasJ0613, modelJ0613_STIG):
     f.fit_toas()
 
 
+def test_SINI_M2():
```

Comment:
Is this testing what happens when only one of SINI/M2 is present and not the other? Shoudl that be two separate tests?